apps/store/public/locales/en/common.json

```
@@ -31,6 +31,7 @@
   "NAVIGATION_STORE_LINK": "Browse all insurances",
   "NAV_MENU_DIALOG_CLOSE": "Close",
   "NAV_MENU_DIALOG_OPEN": "Menu",
+  "PAYMENT_CONNECT_ERROR_LINK_EXPIRED": "The link is has already expired",
```

nit: Wrong in Lokalise 
```suggestion
  "PAYMENT_CONNECT_ERROR_LINK_EXPIRED": "The link has already expired",
```